src/Core/ProjectWizard.php

```diff
@@ -107,7 +107,7 @@ protected function getComposerHttpBasicCredentials()
         $credentials    = [];
         $endString      = '<fg=yellow;options=bold>Composer HTTP-BASIC</> for this project?';
         $questionString = 'Do you want to set '.$endString;
-        while ($this->io->confirm($questionString, false)) {
```

Comment:
> while if mode expert is selected, it will be false as it happens now.
I think changing that would make it true in expert mode as well.

Reply:
sorry, my bad, missed to push my last commit.